pages/index.tsx

```diff
@@ -23,9 +23,26 @@ const versionColors = {
 	'-': 'gray'
 }
 
+const voiceVersionColors = {
+	'-': 'gray',
+	'?': 'red',
+	1: 'red',
+	2: 'red',
+	3: 'red',
+	4: 'yellow',
+	5: 'yellow',
+	6: 'yellow',
+	7: 'yellow',
+	8: 'green',
+	'E2EE': 'green',
+}
+
 const version = (version: number | string) =>
 	<td className={versionColors[version] ?? ((typeof version === 'string' && version.startsWith('9')) ? 'green' : 'yellow')}>{version}</td>
 
+const voiceVersion = (voiceVersion: number | string) => 
+	<td className={voiceVersionColors[voiceVersion] ?? (typeof voiceVersion === 'string' && voiceVersion === 'E2EE' ? 'green' : 'yellow')}>{voiceVersion}</td>
```

Comment:
the fallback logic in `version` is basically a hack to make "9 stable, 10 dev" green instead of yellow. ig a better fallback case would be to look at whatever is before the first space:
```suggestion
const voiceVersion = (voiceVersion: number | string) => 
	<td className={voiceVersionColors[voiceVersion] ?? (typeof voiceVersion === 'string' ? voiceVersionColors[voiceVersion.split(' ')[0]] : null) ?? 'red'}>{voiceVersion}</td>
```
still kinda scuffed but should be more resilient